header.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2009-2016 the original author or authors.
+Copyright © ${project.inceptionYear}-${year} ${owner} (${email})
```

Comment:
Thanks, can you please keep "the original author or authors."? I do not want my name plastered all over the source code.

Reply:
Sure, will do! All of that is just a small preparation for bigger PR incoming :)

Reply:
Done!